app/assets/stylesheets/modules/_uploads.styl

```diff
@@ -90,16 +90,27 @@
     .info
       text-align left
 
-.view-buttons
-  position absolute
-  top 20px
-  right 80px
-
-  button
-    padding 0
-    width 40px
-    height 40px
-    border-radius 50%
+@media (min-width: 721px)
```

Comment:
Is this particular breakpoint essential? We have three other breakpoints that get used elsewhere — 400, 920, and 1200 — that are defined in `_variables.styl` and get used in many stylesheets (eg, see the `+above(desktop)` and `+below(desktop)` parts in `home.styl`. If we can use the same named breakpoints here for consisency, that would be ideal.

Reply:
Actually, I was checking for different devices and small width of browser tab, though `+below(desktop)` looked better choice and as per format when I checked other files. I made a commit with less repetition and no regression in desktop view.